rclpy/src/rclpy/_rclpy.c

```diff
@@ -1161,6 +1161,27 @@ rclpy_get_subscriptions_info_by_topic(PyObject * Py_UNUSED(self), PyObject * arg
   return _get_info_by_topic(args, "subscriptions", rcl_get_subscriptions_info_by_topic);
 }
 
+static PyObject *
+rclpy_get_resolved_subscription_name(PyObject * Py_UNUSED(self), PyObject * args)
```

Comment:
nit, I would name this `rclpy_get_subscription_topic_name`, like `rclpy_get_node_name()` is the `rclpy` wrapper around `rcl_node_get_name()`.